pkg/crd/markers/validation.go

```diff
@@ -53,6 +53,10 @@ var ValidationMarkers = mustMakeAllWithPrefix("kubebuilder:validation", markers.
 	MinLength(0),
 	Pattern(""),
 
+	ItemMaxLength(0),
+	ItemMinLength(0),
+	ItemPattern(""),
```

Comment:
What about XValidation or Enum? And how do we want to handle slice of int for example, min and max values there would also be required for a complete solution.

Wondering if we need to support most of the current validations that can apply to a built-in as items versions as well

Reply:
Yes, for a complete solution most of all scalar markers should have item variant.
If proposed approach is fine in general I can update the PR accordingly.

Reply:
I've updated the PR to support all types of validation markers.